src/ape_geth/provider.py

```diff
@@ -196,11 +204,27 @@ def wait(self, *args, **kwargs):
         self.proc.wait(*args, **kwargs)
 
 
+class _RPCFactory:
+    @cached_property
+    def _default_available(self) -> bool:
+        return _check_url_available(DEFAULT_SETTINGS["uri"])
```

Comment:
suggestion: If we're going to go the HTTP request route, maybe we should at least do a chain_id check to make sure it's the right network?

Reply:
Good point, I have an idea that is much simpler that avoids all this. Brb.